frontend/src/components/Review/index.tsx

```diff
@@ -11,18 +10,32 @@ type Props = {
 };
 
 const Review = ({ overallRating, date, text }: Props): ReactElement => {
+  const formattedDate = format(date, 'MMM dd, yyyy').toUpperCase();
   return (
-    <Card className={styles.card}>
-      <Card.Body>
-        <Card.Title>Anonymous </Card.Title>
-        <Rating rating={overallRating} />
-        <Card.Subtitle className={`${styles.date} mb-2 text-muted`}>
-          {dateToString(date)}
-        </Card.Subtitle>
-        <hr />
-        <Card.Text>{text}</Card.Text>
-      </Card.Body>
-    </Card>
+    <Grid item>
+      <Card>
+        <CardContent>
+          <Grid container spacing={2}>
+            <Grid item container justify="space-between">
+              <Grid container xs={6} spacing={2}>
+                <Grid item>
+                  <HeartRating value={overallRating} readOnly />
+                </Grid>
+                <Grid item>
+                  <Typography>{formattedDate}</Typography>
+                </Grid>
+              </Grid>
+              <Grid item>
+                <Typography variant={'h6'}>Anonymous</Typography>
```

Comment:
for the `'h6'` prop value, since it's just a string, you don't need to use curly brackets